CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Changelog
 
 ### Chores
 
+- Adds dynamic prefix derivation to local storage keys in CardanoNode based on the current network. Adds dynamic derivation of process names for both Daedalus and cardano-node based on the current network. [PR 1109](https://github.com/input-output-hk/daedalus/pull/1109)
```

Comment:
Please remove the mention of Daedalus process names since that is remove now

Reply:
Good catch @DominikGuzei!
I've fixed it now ;)